src/components/WaitForOpponentStep.tsx

```diff
@@ -18,12 +22,10 @@ export default class WaitForOpponentStep extends React.PureComponent {
         </div>
         {selectedPlayId && (
```

Comment:
We can replace `selectedPlayId &&` with `Play[selectedPlayId] &&` on line 18 to deal with the below comment